app/models/device_token.rb

```diff
@@ -0,0 +1,16 @@
+class DeviceToken < ActiveRecord::Base
+  attr_accessible :token
+
+  def save
+    # Escape token: remove arrows and spaces
+    escaped_token = ''
+    token.each_char do |c|
+      escaped_token += c if c != ' ' && c != '<' && c != '>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.